package.json

```
@@ -21,7 +21,8 @@
     "test:watch": "karma start karma.conf.cjs --auto-watch=true --single-run=false",
     "test:update-snapshots": "karma start karma.conf.cjs --update-snapshots",
     "test:prune-snapshots": "karma start karma.conf.cjs --prune-snapshots",
-    "checksize": "rollup -c ; cat my-element.bundled.js | gzip -9 | wc -c ; rm my-element.bundled.js"
+    "checksize": "rollup -c ; cat my-element.bundled.js | gzip -9 | wc -c ; rm my-element.bundled.js",
+    "build": "echo \"This is not a TypeScript project, so no need to build. (You can ignore the guide where it says 'npm run build'.)\""
```

This is genius. 

The text in the Rename section should have been corrected by https://github.com/Polymer/lit-element/pull/1067, but apparently I neglected to merge that, so ... Oops.

Having a deployment issue ATM but will attempt to update the website soon. Meanwhile, I'd like to remove the "ignore the guide" part.

```suggestion
    "build": "echo \"This is not a TypeScript project, so no need to build.\""
```